build-aux/mangen.sh

```diff
@@ -11,4 +11,19 @@ if [ $# -lt 3 ]; then
     exit 1
 fi
 
-sed -e "s|@VERSION[@]|$1|g" $2 >$3
+TMP_FILE=/tmp/mangen.$$
+
+sed -e "s|@VERSION[@]|$1|g" $2 >$TMP_FILE
+
+#
+#  MacOS's groff is missing .UR and .UE support, which makes
+#  URL completely disappear from man pages.  We need to strip
+#  those codes out when building for MacOS
+#
```

Comment:
First. Multiple sed-commands can be added after multiple -e options. 
sed -e s/one/two/g -e s/three/four/g
Second: How about: 
RMURUE=""
if [ $(uname -s) = "Darwin" ] ; then
  RMURUE='-e "s|.UR ||g" -e "s|.UE||g"'
fi
and then the original: 
 sed -e "s|@VERSION[@]|$1|g" $RMURUE $2 >$3

This removes the tmpfile, and the unnecessary copy

Reply:
Quoted arguments with embedded spaces won't parse the way sed expects them to when you do shell variable expansion that way.  (There will be a '`"s|.UR`' argument followed by a '` ||g"`' argument, and sed won't like that.)

I think you'd have to do something like this to get the shell to parse them the way you'd expect:

```
echo sed -e \"s|@Version[@]|$1g\" $RMURUE $2 | sh >$3
```

That seems harder to understand to me, but I can do it that way if you prefer.

Reply:
yea. That stuff I've debugged now and committed and pushed a rewrite. Can you check it works as expected for you? (for debugging I removed the if(darwin).  )

Reply:
Yes, your rewrite works as expected for me.